src/apps/appstore/app.js

```diff
@@ -565,7 +571,17 @@ define(function(require) {
 
 				if (monster.ui.valid(configForm)) {
 					const formData = monster.ui.getFormData('configuration_form');
-					marketAction({ action: 'api_url', api_url: formData.api_url }, 'api_url', saveThis);
+					marketAction(
+						{
+							action: 'update',
+							settings:
+								{
+									api_url: formData.api_ur,
```

Comment:
There seems to be a typo here:
```suggestion
									api_url: formData.api_url,
```